api/store/mongo/device_store.go

```diff
@@ -308,10 +318,23 @@ func (s *Store) DeviceGetByName(ctx context.Context, name, tenant string) (*mode
 }
 
 func (s *Store) DeviceGetByUID(ctx context.Context, uid models.UID, tenant string) (*models.Device, error) {
-	device := new(models.Device)
+	var device *models.Device
+
+	if err := s.cache.Get(ctx, strings.Join([]string{"device", string(uid)}, "/"), &device); err != nil {
+		fmt.Println(err)
```

Comment:
Shouldn't here we return?

Reply:
WIP
https://github.com/shellhub-io/shellhub/blob/94d3a9b492f4599a8dfd95b4668dcd47ccd87b13/api/store/cache/cache_redis.go#L30

Reply:
No really. If `Get` handle the cache failure (and it seems it does) than errors getting here are indeed errors.

Reply:
An error in the cache should not result in application failure.

Reply:
As it is in WIP, a print is enough, when finished it will become just a warning message.